docker/llm/README.md

```diff
@@ -1,148 +1,166 @@
-- [Docker installation guide for IPEX LLM Fine Tuning on CPU](#docker-installation-guide-for-ipex-llm-fine-tuning-on-cpu) 
-- [Docker installation guide for IPEX LLM Fine Tuning on XPU](#docker-installation-guide-for-ipex-llm-fine-tuning-on-xpu) 
+- [Docker Installation](#docker-installation-instructions)
+- [IPEX-LLM Inference](#ipex-llm-inference-on-cpu)
+    - [On CPU](#ipex-llm-inference-on-cpu)
+    - [On XPU](#ipex-llm-inference-on-xpu)
+- [IPEX-LLM Serving](#ipex-llm-serving-on-cpu)
+    - [On CPU](#ipex-llm-serving-on-cpu)
+    - [On XPU](#ipex-llm-serving-on-xpu)
+- [IPEX-LLM Fine Tuning](#ipex-llm-fine-tuning-on-cpu)
+    - [On CPU](#ipex-llm-fine-tuning-on-cpu)
+    - [On XPU](#ipex-llm-fine-tuning-on-xpu)
 
-## Docker installation guide for IPEX-LLM on CPU
 
-### IPEX-LLM on Windows
+## Docker Installation Instructions
 
-#### Install docker
+1. **For New Users:**
+   - Begin by visiting the [official Docker Get Started page](https://www.docker.com/get-started/) for a comprehensive introduction and installation guide.
 
-New users can quickly get started with Docker using this [official link](https://www.docker.com/get-started/).
+2. **Additional Steps for Windows Users:**
+   - For Windows Users, follow the step-by-step guide: [Docker Installation Instructions for Windows](https://github.com/intel-analytics/ipex-llm/blob/main/docs/readthedocs/source/doc/LLM/Quickstart/docker_windows_gpu.md).
 
-For Windows users, make sure WSL2 or Hyper-V is enabled on your computer. 
-The instructions for installing can be accessed from 
-[here](https://docs.docker.com/desktop/install/windows-install/).
 
+## IPEX-LLM Inference on CPU
 
-#### Pull ipex-llm-cpu image
+### 1. Prepare ipex-llm-cpu Docker Image
 
-To pull image from hub, you can execute command on console:
+Run the following command to pull image from dockerhub:
 ```bash
-docker pull intelanalytics/ipex-llm-cpu:2.5.0-SNAPSHOT
-```
-to check if the image is successfully downloaded, you can use:
-```powershell
-docker images | sls intelanalytics/ipex-llm-cpu:2.5.0-SNAPSHOT
+docker pull intelanalytics/ipex-llm-cpu:2.1.0-SNAPSHOT
 ```
 
+### 2. Start bigdl-llm-cpu Docker Container
 
-#### Start ipex-llm-cpu container
-
-To run the image and do inference, you could create and run a bat script on Windows.
-
-An example on Windows could be:
-```bat
-@echo off
-set DOCKER_IMAGE=intelanalytics/ipex-llm-cpu:2.5.0-SNAPSHOT
-set CONTAINER_NAME=my_container
-set MODEL_PATH=D:/llm/models[change to your model path]
+```bash
+#/bin/bash
+export DOCKER_IMAGE=intelanalytics/ipex-llm-cpu:2.1.0-SNAPSHOT
+export CONTAINER_NAME=my_container
+export MODEL_PATH=/llm/models[change to your model path]
 
-:: Run the Docker container
-docker run -itd ^
-    -p 12345:12345 ^
-    --cpuset-cpus="0-7" ^
-    --cpuset-mems="0" ^
-    --memory="8G" ^
-    --name=%CONTAINER_NAME% ^
-    -v %MODEL_PATH%:/llm/models ^
-    %DOCKER_IMAGE%
+docker run -itd \
+    --privileged \
```

Comment:
is privileged mode necessary?

Reply:
When executing the command `numactl -C 0-47 -m 0 python $(dirname "$0")/run.py`, the Docker container needs to have privileged mode enabled.